pages/faqs/yes-you-can-faq.mdx

```diff
@@ -97,6 +97,10 @@ Yes, you can! We integrate directly with Segment Protocols. <Link passHref href=
 
 Yes, you can! We integrate directly with RudderStack. <Link passHref href="/workspace/tracking-plan/publishing#rudderstack"><a>Find out more here</a></Link>.
 
+### Can I publish my Avo tracking plan to mParticle?
+
+Yes, you can! We integrate directly with mParticle. <Link passHref href="/workspace/tracking-plan/publishing#mparticle"><a>Find out more here</a></Link>.
```

Comment:
```suggestion
Yes, you can! We integrate directly with mParticle Data Master. <Link passHref href="/workspace/tracking-plan/publishing#mparticle"><a>Find out more here</a></Link>.
```

Reply:
I feel like they might change this name at some point